server/router.go

```diff
@@ -44,7 +44,7 @@ func (r *Router) GenerateProjectJobURL(ctx models.ProjectCommandContext) (string
 	pull := ctx.Pull
 	projectIdentifier := models.GetProjectIdentifier(ctx.RepoRelDir, ctx.ProjectName)
 	jobURL, err := r.Underlying.Get(r.ProjectJobsViewRouteName).URL(
-		"org", pull.BaseRepo.Owner,
+		"org", strings.ReplaceAll(pull.BaseRepo.Owner, "/", "-"),
```

Comment:
Isn't this all you need?

Reply:
I stated with just this change first, and I've seen just empty screen in console because Atlantis creates output steam mapping for this URL. Output handler uses code in models.go and pull_closed_executor.go to calculate output map key.

Reply:
Yup, that makes sense! Thanks for the fix @PertsevRoman :)

Reply:
Sure, thanks for review guys!

Reply:
@nishkrishnan do you have any question ? Could I resolve the conversation ?

Reply:
sounds good